apps/core/components/passengerForm/PassengerForm.js

```diff
@@ -48,23 +50,39 @@ const nationalityData = [
   },
 ];
 
+const initialFormState = {
+  gender: 'male',
+  name: null,
+  lastName: null,
+  id: null,
+  nationality: null,
+  dateOfBirth: null,
+  bags: null,
+  passengerCount: 1,
+};
+
 type State = {
   ...PassengerCardType,
   lastName: ?string,
+  hasPrefilled: boolean,
 };
 
 class PassengerForm extends React.Component<Props, State> {
   state = {
-    gender: 'male',
-    name: null,
-    lastName: null,
-    id: null,
-    nationality: null,
-    dateOfBirth: null,
-    bags: null,
-    passengerCount: 1,
+    ...initialFormState,
+    hasPrefilled: false,
   };
 
+  static getDerivedStateFromProps(props, state) {
+    // Reset the form if we add a passenger after an edit process
```

Comment:
Looks like reset doesn't always behave correctly. When I try to add second user in row, form is not cleared.

![pass-save](https://user-images.githubusercontent.com/2660330/57861114-57310000-77f6-11e9-9b4c-ffc64d7ad503.gif)

Reply:
Good catch! will look at it now

Reply:
Fixed!

Reply:
👍Problem is gone, thanks!